### ISSUE TITLE
No Variable Replacement in graphQl Request Body

### ISSUE THREAD

**Issue by Andreas-Schoenefeldt** [usebruno/bruno#3210]
### I have checked the following:

- [X] I use the newest version of bruno.
- [X] I've searched existing issues and found nothing related to my issue.

### Describe the bug

I can not tell when it stopped working, but Variables in graphQl request are no longer replaced for me (currently on v1.30.1 on Mac OS, at least from 1.28.0).

This is my request:

Request Script:
```
var timestamp = new Date().getTime();
bru.setVar("TIMESTAMP", timestamp);
```

Query:
```graphQl
mutation addCategory($data: AddCategoryInput!) {
  addCategory(input: $data) {
    ...CategoryFragment
    __typename
  }
}

fragment CategoryFragment on Category {
  id
  name
  parentId
  parentType
  __typename
}
```

Variables:
```json
{
  "data": {
    "name": "{\"en\":\"Category - {{TIMESTAMP}}\"}",
    "parentType": "Project",
    "parentId": "8"
  }
}
```

`{{TIMESTAMP}}` used to be correctly replaced by a timestamp, now it is not anymore:

```
> POST https://127.0.0.1:8000/graphql/graphql/app
> X-AUTH-TOKEN: xyz
> content-type: application/json
> cookie: PHPSESSID=123
> data
{
  "query": "mutation addCategory($data: AddCategoryInput!) {\n  addCategory(input: $data) {\n    ...CategoryFragment\n    __typename\n  }\n}\n\nfragment CategoryFragment on Category {\n  id\n  name\n  parentId\n  parentType\n  __typename\n}\n",
  "variables": {
    "data": {
      "name": "{\"en\":\"Category - {{TIMESTAMP}}\"}", 
      "parentType": "Project",
      "parentId": "8"
    }
  }
}
```

### Screenshots/Live demo link

<img width="1387" alt="Screenshot 2024-09-27 at 10 28 09" src="https://github.com/user-attachments/assets/93388082-e025-4f7c-b8f6-0b1b9f42e4ec">

**Comment by Citizen4our** [comment]
Hello, I have the same bug.
It worked on 1.26.2 version, from 1.27 - no.
Video with versions:
1.26.2
1.27
1.30.1

https://github.com/user-attachments/assets/68072259-4bcd-43b4-9387-b6ec0538a76e

**Comment by Andreas-Schoenefeldt** [comment]
It woulb be good, if the testcoverage would prevent a bug in such an important core functionality. In my opinion this is a blocker issue.

**Comment by helloanoop** [comment]
Just merged the PR - #3212 for the fix. This will be available in the upcoming v1.32.0 release/

**Comment by Andreas-Schoenefeldt** [comment]
Thank you! ❤️
